go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/BuxOrg/bux-server
 go 1.21.5
 
 require (
-	github.com/BuxOrg/bux v0.14.3-0.20240201141930-5ac2858facdc
+	github.com/BuxOrg/bux v0.14.4-0.20240212060332-36a09b80059f
```

Comment:
I used a bux branch [feat-567-metrics](https://github.com/BuxOrg/bux/tree/feat-567-metrics) as a temporary dependency. 
*Remember* to switch it after this branch is merged to `rc-v1.1`.